docs/pages/versions/unversioned/config/package-json.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: package.json
+description: A reference for Expo specific properties that can be used in the package.json file.
```

Comment:
```suggestion
description: A reference for Expo-specific properties that can be used in the package.json file.
```
I think the hyphen generally applies when another word modifies a noun to function as an adjective, but it sounds like maybe it's not 100% hard-and-fast? https://www.grammarly.com/blog/punctuation-capitalization/hyphen/

Reply:
It's not, but let's go with it.